component/prometheus/scrape/scrape.go

```diff
@@ -81,6 +80,8 @@ type Arguments struct {
 
 	// Scrape Options
 	ExtraMetrics bool `river:"extra_metrics,attr,optional"`
+
+	ClusteringEnabled bool `river:"clustering_enabled,attr,optional"`
```

Comment:
I would like to see this as a block rather than an attribute, even if it's just a block with a single attribute in it:

```river 
clustering {
  enabled = true 
}
```

The main reason I want to do this is because we'll probably document a lot of information relevant to clustering, and it would be nice to document that in its own dedicated section for the `clustering` block, whereas right now it would have to be all jammed into the arguments block documentation.